src/domains/canoneunico/03_storage_account.tf

```diff
@@ -0,0 +1,65 @@
+module "cu_sa" {
```

Comment:
is possible to use a better name for this resource id?

Reply:
done